### DISCUSSION TITLE
Please help me.

### DISCUSSION THREAD

**Discussion by zhao605207225** [community/community#48843] [GitHub Education] [answered]
About Student Authentication,
I've tried to verify it a dozen times,
Attempted to upload my admission notice, curriculum, transcripts, payment slip, but still failed certification,
Does it have other auditing methods?
such as manual.
![1677617604386](https://user-images.githubusercontent.com/75011317/221977320-a5cdfc7e-383a-4c80-9bd6-6e29e86df018.jpg)

**Comment by vishnubansal001** [comment]
If you have attempted to upload all the required documents for certification and still failed, there may be other auditing methods that you can try. Here are some suggestions:

1. Check the requirements: Make sure that you have provided all the required documents and that they meet the specific requirements for the certification. Check the certification website or contact the certifying organization to confirm the requirements.
2. Double-check your documents: Ensure that your documents are legible, up-to-date, and complete. Make sure that all the required information is included and that your documents are in the correct format.
3. Seek assistance: If you are still having trouble, reach out to the certification organization or the relevant authority for assistance. They may be able to provide additional guidance or support.
4. Consider an appeal: If you have been denied certification, you may be able to appeal the decision. Check the certification website or contact the certifying organization to find out the appeal process.
5. Try other certifications: If you are unable to obtain the desired certification, consider exploring other certifications in your field. There may be alternative certifications that are easier to obtain or that better match your qualifications and experience.

Remember, the certification process can be complex and may require patience and persistence. Keep trying, and seek help when needed.

**Comment by Qdog31** [comment, marked as answer]
One of the maintainers made a post about FAQs that may be able to help you. 

Original article: https://github.com/orgs/community/discussions/17814
